src/OpenApi/test/Microsoft.AspNetCore.OpenApi.Tests/Extensions/ApiDescriptionExtensionsTests.cs

```diff
@@ -31,17 +31,25 @@ public void MapRelativePathToItemPath_ReturnsItemPathForApiDescription(string re
         Assert.Equal(expectedItemPath, itemPath);
     }
 
+    public static class HttpMethodTestData
+    {
+        public static IEnumerable<object[]> TestCases => new List<object[]>
```

Comment:
Regarding my comment about QUERY, is it worth adding a reflection-based test that checks for `HttpMethod` having any new static property to flush out the need to react to any addition?

Reply:
I think we can probably forgo on this until the spec supports more HTTP methods. From my understanding, moving from OperationType to HttpMethod in this breaking change was a setup for that.